globalConfig.json

```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:53561/jest?","mongoDBName":"jest"}
\ No newline at end of file
+{"mongoUri":"mongodb://127.0.0.1:51647/jest?","mongoDBName":"jest"}
```

Comment:
What is this for?

Reply:
This file was modified after running the code base's test suite. It seems to have created an in-memory mongodb server for test purposes. I will add to gitignore, as it seems to create/modfy the config file every time the tests are run.